pkgs/tools/misc/spacer/default.nix

```diff
@@ -5,16 +5,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "spacer";
-  version = "0.1.7";
+  version = "0.1.8";
 
   src = fetchFromGitHub {
     owner = "samwho";
     repo = "spacer";
     rev = "v${version}";
-    hash = "sha256-eFXxcOhUqyo3eUws3RCO0w+0XGlxZAonKFTphnrhHs8=";
+    hash = "sha256-F2PrBlS9uL6BQtjNNabCpl3ofavsUGlBy/Hotm42Oec=";
   };
 
-  cargoHash = "sha256-z7igJc8HHqpiY2an4hFWoZElwgF5NUA+TFPqxuowC/w=";
+  cargoHash = "sha256-aeXvqgSKRvm9W6sc5XCkwhMYUncj8pEPyQpYQr+fj7Y=";
 
   # Cargo.lock is outdated
   preConfigure = ''
```

Comment:
I think this `preConfigure` is not needed anymore. Would you mind removing it?

Reply:
good catch!